sormas-ui/src/main/java/de/symeda/sormas/ui/utils/AbstractDetailView.java

```diff
@@ -115,4 +115,5 @@ protected R getReference() {
 	 *            The URL parameters String
 	 */
 	protected abstract void initView(String params);
+
```

Comment:
Why empty line here? It's against code convention

Reply:
fixed